src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -795,6 +795,9 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ResolvedFileToPublish Include="@(_RazorDebugSymbolsIntermediatePath)" Condition="'$(CopyOutputSymbolsToPublishDirectory)'=='true'">
         <RelativePath>@(_RazorDebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
+
+      <!-- Pass assembly to linker -->
+      <TrimmerAdditionalAssembly Include="@(RazorIntermediateAssembly)" Condition="'$(CopyBuildOutputToPublishDirectory)'=='true'">
```

Comment:
Missed closing the tag. The change looks good, but I wonder if it'll break all our functional tests since it's running against an ancient build of the SDK.

Reply:
Yeah, I was copying my change to a franken build and fixed the typo in the copy butnot the source.